README.md

```diff
@@ -2,31 +2,32 @@
 
-We want to ensure that you are as prepared as possible for the technologies and concepts that your instruction staff will be coaching you through in the first few weeks, so we have prepared this couse to set you up for success. These resources will provide you with the basic skills, terminology, and context for what it will be like at Nashville Software School.
+We want to ensure that you are as prepared as possible for the technologies and concepts that your instruction staff will be coaching you through in the first few weeks, so we have prepared this course to set you up for success. These resources will provide you with the basic skills, terminology, and context for what it will be like at Nashville Software School.
 
 ## Foundations Course is Mandatory
 
-If you don't do the **Foundations Course**, you will be immediately behind your teammates on the first day of class and are at risk of needing to move to the next cohort so that you have time to complete it.
+If you do not do the **Foundations Course**, you will be immediately behind your teammates on the first day of class and are at risk of needing to move to the next cohort so that you have time to complete it.
 
-Nashville Software School is an accelerated bootcamp, and you will hit the ground running on day one. We will assume that you've completed this work, and provide minimal resources to you if you have not. Additionally, the life of a software developer is one of focus and meeting committments. Completing the **Foundations Course** shows us that you have the ability to prioritize this work, can focus on it, and meet the committment of completing it.
+Nashville Software School is an accelerated Bootcamp, and you will hit the ground running on day one. We will assume that you have completed this work and provide minimal resources if you have not. Additionally, the life of a software developer is one of focus and meeting commitments. Completing the **Foundations Course** shows us that you can prioritize this work, focus on it, and meet the commitment of completing it.it.
 
 ## Foundation Sessions
 
-Another opportunity we provide for your foundations are training sessions before your cohort starts. Starting 4 weeks before the beginning of the cohort, on each Thursday evening, you can come in to ask questions, meet your teammates, and get some mentoring on this **Foundations Course**. These are a great opportunity for you to make sure you are set up for success on day one of your cohort.
+Another opportunity we provide for your Foundations is coaching sessions every Monday evening. Starting approximately four weeks before your cohort begins, you can come to ask questions, meet your teammates, and get some mentoring from NSS Alumni and Junior Instructors on this **Foundations Course**.
```

Comment:
I think `Foundations` should be lowercase here. It is not referring to the title of the course, just an individual's foundational skills. 

I think that we may change the days/times of sessions depending on what is best. Can we say `Another opportunity we provide to reinforce your software development foundations is coaching sessions that start approximately four weeks before your cohort begins. You can come to ask questions, meet your teammates, and get some mentoring from NSS alumni and junior instructors.`